tutorial/en/build-run.md

```diff
@@ -9,8 +9,8 @@ If you want to build the `nameservice` application in this repo to see the funct
 make install
 
 # Now you should be able to run the following commands:
-nsd help
-nscli help
+./nsd help
```

Comment:
`nsd` and `nscli` would be create in folder`${GOBIN}`. 
After the following steps, `${GOBIN}` was already appended to ${PATH}. 
So, we can run `nscli` directly, `./` is redundancy

```bash
mkdir -p $HOME/go/bin
echo "export GOPATH=$HOME/go" >> ~/.bash_profile
echo "export GOBIN=\$GOPATH/bin" >> ~/.bash_profile
echo "export PATH=\$PATH:\$GOBIN" >> ~/.bash_profile
echo "export GO111MODULE=on" >> ~/.bash_profile
source ~/.bash_profile
```